.vscode/settings.json

```diff
@@ -1,28 +1,10 @@
 {
     "editor.tabSize": 2,
-
-    "[javascript]": {
-        "editor.defaultFormatter": "esbenp.prettier-vscode",
-        "editor.formatOnSave": true
-    },
-    "[typescript]": {
-        "editor.defaultFormatter": "esbenp.prettier-vscode",
-        "editor.formatOnSave": true
-    },
```

Comment:
Why'd we remove all this?

Reply:
I was cleaning out the deprecated fields and accidentally removed those two also. Placed it back in, good catch.

Reply:
I still see some fields as missing, relying on @hbergren and you to do the right thing.

Reply:
hmm it shows up for me when on the files view in the PR:

https://github.com/xpring-eng/Xpring-JS/pull/138/files#diff-5b46cfda0ddaed371f52aa24ea2b0b2aL4